docs/project/psa2.md

```diff
@@ -0,0 +1,301 @@
+# Project Start Architectuur voor Expertteam Digitale Toegankelijkheid, onderdeel webredactie
+
+https://www.noraonline.nl/wiki/Webversie_PSA_Format#1_Managementsamenvatting_(max_1_A4)
+
+## Status
+
+Status: Concept.
+
+Datum: april 2025.
+
+## 1. Managementsamenvatting
+
+De Nederlandse overheid moet voldoen aan standaarden voor digitale toegankelijkheid, daarom moeten overheidsinstellingen problemen met toegankelijkheid voorkomen of oplossen. De naleving wordt getoetst met periodieke onderzoeksrapporten. Daaruit blijkt dat content van websites vaak niet voldoende toegankelijk is.
+
+Content meer toegankelijk maken kan op twee directe manieren: contentmakers ondersteunen bij toegankelijke content maken, en door voorkomen dat in een webpagina's met toegankelijkheidsproblemen worden gepubliceerd. <!-- TODO: Moeten we nog de organisatorische aanpak noemen, en verwijzen naar andere projecten diet dat doen? Zoals: mandaat om "nee" te zeggen tegen ontoegankelijke content die wordt aangeleverd. -->
+
+We zoeken een oplossing waarmee de belangrijkste problemen in content voorkomen en opgelost kunnen worden. Contentmakers zijn vaak niet op de hoogte van problemen, terwijl die in veel gevallen automatisch herkend kunnen worden, waardoor publicatie van ontoegankelijke content niet wordt voorkomen. Sommige toegankelijkheidsproblemen kunnen niet automatisch herkend worden, maar die moeten opgelost door extra stappen in het publicatieproces. We zoeken daarom een oplossing die waar mogelijk problemen herkent en voorkomt, en waar nodig de redactie helpt door instructies en documentatie in het proces te verwerken.
+
+<!-- De belangrijkste bestaande kaders en inrichtingskeuzes waarmee rekening moet worden gehouden. Benoem hierbij ook de samenhang met architecturen. Bijvoorbeeld door aan te geven welk deel van een enterprise-, domein- of ketenarchitectuur de verandering invulling geeft. -->
+
+De effectiviteit van de oplossing is afhankelijk van goede integratie met de software die gebruikt wordt door de webredactie: de CMS-software van bestaande websites. Bij de overheid zijn 3 CMS softwarepakketten de grootste: Drupal, WordPress, TYPO3. De oplossing zou zoveel mogelijk generiek moeten zijn, met voor de belangrijkste CMS een plugin. <!-- warning: we beloven nu 3 plugins met de volledige functionaliteit -->
```

Comment:
Moeten we hier nog Open Source pakketten specificeren. Ik weet namelijk niet zeker of Typo3 het wint als we dat niet doen :) En volgens mij hadden we ook bedacht om top 2 te doen omdat we weinig draagvlak hebben gevonden voor Typo3?